agreements/sponsorship/sponsorship-agreement.md

```diff
@@ -57,6 +61,7 @@ Make your sponsorship payment directly through [Open Collective][]:
 
 - Payment is charged annually
 - Modify or cancel anytime through your Open Collective account
+- Select the Commonhaus Foundation or a specific Project when making payment
```

Comment:
```suggestion
- Select the Commonhaus Foundation or a specific Project(s) when making payment
```

Not sure if this is the best way to indicate it, but do we need to specify here it can be more than one project? Below there are certainly multiple possible lines. Should we make it explicit in wording?

Reply:
no. Not with OpenCollective... You choose one. I mean, they could choose more than one, but they create the contribution once per selected place. I removed `a`, which I think fixes it.